cristin-commons/src/main/java/no/unit/nva/exception/HttpClientFailureException.java

```diff
@@ -0,0 +1,27 @@
+package no.unit.nva.exception;
+
+
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import org.apache.http.HttpStatus;
+
+public class HttpClientFailureException extends ApiGatewayException {
+
+    public static final int ERROR_CODE = HttpStatus.SC_BAD_GATEWAY;
```

Comment:
better to use HttpUrlConnection to avoid using the apache library only for the error codes.